src/scrabble.js

```diff
@@ -1,6 +1,111 @@
 class Scrabble {
-  // Write your implementation here
+  constructor (word) {
+    this.word = word
+    this.wordMultiply = word
+    this.scoreMap = {
+      // score = 1
+      A: 1,
+      E: 1,
+      I: 1,
+      O: 1,
+      U: 1,
+      L: 1,
+      N: 1,
+      R: 1,
+      S: 1,
+      T: 1,
+      // score = 2
+      D: 2,
+      G: 2,
+      // score = 3
+      B: 3,
+      C: 3,
+      M: 3,
+      P: 3,
+      // score = 4
+      F: 4,
+      H: 4,
+      V: 4,
+      W: 4,
+      Y: 4,
+      // score = 5
+      K: 5,
+      // score = 8
+      J: 8,
+      X: 8,
+      // score = 10
+      Q: 10,
+      Z: 10,
+      // score = 0
+      '{': 0,
+      '}': 0,
+      '[': 0,
+      ']': 0,
+      ',': 0
+    }
+  }
 
+  doubleLetter () {
+    const wordArray = []
+    for (let i = 0; i < this.word.length - 1; i++) {
+      if (this.word.charAt(i) === '{' && this.word.charAt(i + 2) === '}') {
+        wordArray.push(this.word.charAt(i + 1))
+      }
+    }
+    return wordArray.toString() ?? '' // nullish coallescing - return RHS when LHS is falsy
```

Comment:
This nullish coalescing is not required. When you use `toString()` on an empty array it will return `''`